keps/sig-contributor-experience/1553-issue-triage/README.md

```diff
@@ -248,23 +242,24 @@ Which leaves the following label and accompanying definition:
 
 ## Drawbacks
 
-N/A
-
-<<[UNRESOLVED]>>
+- It is yet-another-label for our contributors to keep track of.
+- A new contributor may not understand why it is applied and who can resolve it.
 
 ## Alternatives
 
-<!--
-What other approaches did you consider and why did you rule them out?  These do
-not need to be as detailed as the proposal, but should include enough
-information to express the idea and why it was not acceptable.
--->
-
-<<[/UNRESOLVED]>>
+Potential workflows could be built with the current set of labels in use. For
+example, issue triage teams could apply `triage/unresolved` to open issues that
+have not yet been acted upon. This process is done currently for `sig/network`
+issues automatically by [Athenabot].
```

Comment:
Just as a side note, removal of `triage/unresolved` would break Athenabot. The bot could continue adding comments with `/triage unresolved`, but no label change would take place.

We should ensure that we explicitly communicate with sig-network before rolling this out.

Reply:
i've noticed that in some cases the bot also automatically applies `/sig ...` based on content.

Reply:
cc @vllry